Swiftfin tvOS/Views/HomeView/HomeView.swift

```diff
@@ -34,5 +34,11 @@ struct HomeView: View {
         }
         .edgesIgnoringSafeArea(.top)
         .edgesIgnoringSafeArea(.horizontal)
+        .onAppear(perform: {
```

Comment:
Remove based on comment in the view model. Additionally, this is attached to a `Group` which would require more testing when changing states since I think the `onAppear/onDisappear` would be in an undeterministic with all of the views.